userguide/sharing.rst

```diff
@@ -2697,3 +2541,202 @@ Go back to
 --> View File Extents` and click the :guilabel:`Edit` button for the
 file extent. Set the size to *0* as this causes the iSCSI target to
 use the new size of the file.
+
+
+.. index:: Time Machine
+.. _Creating Authenticated and Time Machine Shares:
+
+Creating Authenticated and Time Machine Shares
+----------------------------------------------
+
+macOS includes the Time Machine feature which can be used to perform
+automatic back ups. %brand% supports Time Machine backups for both
+:ref:`AFP <Apple (AFP) Shares>` and :ref:`SMB <Windows (SMB) Shares>`
+shares. This section has instructions to create Time Machine AFP and SMB
+shares, using the
+:menuselection:`Wizard`
+to create an AFP Time Machine share, configuring a Time Machine share.
+The process for creating an authenticated share for a user is the same
+as creating a Time Machine share for that user.
+
+
+Manual Creation of Authenticated or Time Machine Shares
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Both Time Machine and authenticated shares should be created on an
```

Comment:
Make this imperative.
"Create Time Machine or authenticated shares on a new dataset."